src/Common/src/TypeSystem/Common/TypeSystemException.cs

```diff
@@ -193,6 +193,10 @@ public static string OwningModule(MethodDesc method)
 
             public static string OwningModule(TypeDesc type)
             {
+#if TYPE_LOADER_IMPLEMENTATION
```

Comment:
At some point, my implementation didn't have the ifdef. That was what @davidwrighton suggested I do :)